packages/@aws-cdk/cloud-assembly-schema/scripts/update-schema.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 scriptsdir=$(cd $(dirname $0) && pwd)
-packagedir=$(realpath ${scriptsdir}/..)
+packagedir=$(cd ${scriptsdir}/.. && pwd)
```

Comment:
@iliapolo FYI (does not work on a mac)

Reply:
That’s...weird...pretty sure it worked on my Mac 🤷‍♂️

Reply:
you may have `realpath` installed.

Reply:
https://github.com/whatwg/html-build/issues/90#:~:text=realpath%20is%20not%20present%20by,Closes%20%2390.&text=realpath%20is%20not%20present%20by%20default%20on%20macOS%2C%20and%20on,not%20support%20%2D%2Drelative%2Dto.